subsys/bluetooth/host/conn.c

```diff
@@ -2170,6 +2170,10 @@ int bt_conn_le_start_encryption(struct bt_conn *conn, uint8_t rand[8],
 	struct bt_hci_cp_le_start_encryption *cp;
 	struct net_buf *buf;
 
+	if (len > sizeof(conn->ltk)) {
```

Comment:
```suggestion
	if (len > sizeof(cp->ltk)) {
```

Suggest to use the value of the where the `ltk` is actually being copied to

Reply:
yep, just saw it. I think it is better to check it directly from the struct because if this value changes we don't need to change it here.